### ISSUE TITLE
Movement Tracker - Doesn't recognize non-walking movements

### ISSUE THREAD

**Issue by nlblunt** [theripper93/enhancedcombathud#40]
The movement tracker does not recognize movement types other than walking. If a creature only has Flying, for example, the movement tracker shows 0 of 0 movement.

**Comment by theripper93** [comment]
yep, there are no plans to add a switch or something similar, could grab the highest one tho

**Comment by theripper93** [comment]
in next version, the larger value will be used
